src/topwidget.cpp

```diff
@@ -43,6 +45,8 @@ TopWidget::TopWidget(QWidget *parent) :
     addAction(menuAction);
     setContextMenuPolicy( Qt::PreventContextMenu );
 
+    connect(KiwixApp::instance(), &KiwixApp::currentTitleChanged,
+            this, &TopWidget::updateBackForwardButtons);
```

Comment:
I wonder if we really want to connect to the `currentTitleChanged` ?
The title will change if we change page (and we need to update buttons) but it will also change if the html tell it so (in instance, if I remember correctly, the title is changed twice when we change page. One first time with the "path" of the page and a second time with the title after the html is parsed).

In the same time, `updateBackForwardButtons` is triggered by this event but it doesn't rely on it. It ask for the real history to update the button. So it may be not so important.